includes/calendars/views/default-calendar-grid.php

```diff
@@ -264,11 +264,16 @@ class="simcal-nav simcal-current"
                 </tr>
                 </thead>
 
-				<?php echo $this->draw_month(date('n', $calendar->start), date('Y', $calendar->start)); ?>
-
+				<?php echo $this->draw_month(date('n', $calendar->start), date('Y', $calendar->start)); ?>	 			
             </table>
 
-			<?php echo '<div class="simcal-ajax-loader simcal-spinner-top" style="display: none;"><i class="simcal-icon-spinner simcal-icon-spin"></i></div>';}
+			<?php
+   $is_print_calendar = get_post_meta($calendar->id, '_display_print_calendar');
+   if ($is_print_calendar[0] === 'yes') {
+   	echo '<button id="print-calendar-button" class="print-calendar-button">Print Calendar</button>';
+   }
+   echo '<div class="simcal-ajax-loader simcal-spinner-top" style="display: none;"><i class="simcal-icon-spinner simcal-icon-spin"></i></div>';
```

Comment:
What is this for?

Reply:
This is for loader but now I removed it because this is no need of loaders, in print option